go.mod

```diff
@@ -5,7 +5,10 @@ go 1.22
 toolchain go1.22.0
 
 require (
-	github.com/Layr-Labs/eigenda v0.8.4
+	// pointing to latest commit of https://github.com/Layr-Labs/eigenda/pull/821
+	// TODO: merge eigenda-client PR first, and then point to the latest master commit
+	//       instead of PR commit (which will get squash and merged)
+	github.com/Layr-Labs/eigenda v0.8.5-0.20241025110857-96ad889d3c17
```

Comment:
TODO: merge https://github.com/Layr-Labs/eigenda/pull/821 before this PR

Reply:
Also notttt....... sure why the ci workflows are all broken. Can't find batchHeader... at lines that don't even exist. lint test and build work fine locally for me so I'm confused. Wondering if it doesn't have to do with this reference somehow..

Reply:
merged + updated: [6a25446](https://github.com/Layr-Labs/eigenda-proxy/pull/192/commits/6a25446508b5e9e4a89f2fabd0027d1b7f739e4b)